framework/include/ble_manager/ble_scan.h

```diff
@@ -64,8 +64,18 @@ typedef struct {
 
 /* This is a heuristic value. It can be customized following a system env */
 #define SCAN_INFO_BUFFER_SIZE 100
+
+#ifdef CONFIG_BLE_SCAN_MAX_WHITELIST
+#define SCAN_WHITELIST_SIZE CONFIG_BLE_SCAN_MAX_WHITELIST
+#else
```

Comment:
How about to gather all BLE configuration in one file(ex. ble_config.h).
Then it's more easy to handle ble configurations.